bot/cogs/cyber.py

```diff
@@ -81,7 +81,7 @@ def __init__(self, bot: Bot):
         with open("assess.json") as f:
             assess_docs = load(f)
 
-        if challenge_num not in range(len(assess_docs) + 1):
+        if challenge_num not in range(len(assess_docs) + 1) and challenge_num != 0:
```

Comment:
On second thoughts, why are we even calling the `range` function in the first place?
```suggestion
        0 < challenge_num <= len(access_docs):
```